canvas_modules/common-canvas/src/common-canvas/canvas-controller.js

```diff
@@ -495,6 +495,11 @@ export default class CanvasController {
 		this.objectModel.clearNotificationMessages();
 	}
 
+	// Removes the notification message(s) with the given id(s)
```

Comment:
Can we mention that ids is an array of IDs ?
BTW - What I've done for CanvasController API documentation is to take the method prologs and method signature from the code (like this one) and copy them into the Wiki. So hopefully you can do the same with this method prolog and method signature.

Reply:
updated the description comment. I'll copy over the signature and explain in the wiki as well when merged